utils/utils/src/main/scala/pl/touk/nussknacker/engine/util/json/BestEffortJsonEncoder.scala

```diff
@@ -27,14 +27,20 @@ case class BestEffortJsonEncoder(failOnUnkown: Boolean, classLoader: ClassLoader
   private val safeInt = safeJson[Int](fromInt)
   private val safeDouble = safeJson[Double](fromDoubleOrNull)
   private val safeFloat = safeJson[Float](fromFloatOrNull)
-  private val safeBigDecimal = safeJson[java.math.BigDecimal](a => fromBigDecimal(a))
   private val safeBigInt = safeJson[java.math.BigInteger](a => fromBigInt(a))
   private val safeNumber = safeJson[Number](a => fromDoubleOrNull(a.doubleValue())) // is it correct?
 
-  val circeEncoder: Encoder[Any] = Encoder.encodeJson.contramap(encode)
+  private val safeBigDecimal = safeJson[java.math.BigDecimal] { v =>
```

Comment:
Let's add a comment that while it's technically valid WRT e.g. integer JSONSchema, we do it for better readability (or personal preference...)